sdks/wasm/binding_support.js

```diff
@@ -94,7 +96,7 @@ var BindingSupportLib = {
 			this.object_to_string = get_method ("ObjectToString");
 
 			this.object_to_enum = get_method ("ObjectToEnum");
-
+			BINDING.mono_wasm_core_map = new Map();
```

Comment:
I think this should probably be a WeakMap

Reply:
Addressed in recent commit